util/internal/integrationtest/v5migrationhelper/src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,40 @@
+### direct log messages to stdout ###
```

Comment:
You can safely remove this file. This module depends on the main test utils module, which already provides Log4j configuration, and while different, that configuration is enough for our needs here.

Reply:
Since there was a `log4j.properties` here, I'll do in a different commit.

Reply:
My point was, you changed it already, and what's more, you changed it to match the main code...